### PR TITLE
feat: add enable_turbo variable

### PR DESCRIPTION
**Pull request by Ruesa18**
Introduce new `enable_turbo` variable to control whether or not to use turbo on tables and other content

### CODE REVIEW

File: src/DependencyInjection/araiseTableExtension.php [hunk 1]
```diff
@@ -6,35 +6,25 @@
...
+    public function load(array $configs, ContainerBuilder $container): void
     {
-        if (! $container->hasExtension('doctrine_migrations')) {
-            return;
-        }
+        $configuration = new Configuration($container);
+        $config = $this->processConfiguration($configuration, $configs);
+
+        $container->setParameter('araise_table.enable_turbo', $config['enable_turbo']);
+        $container->setParameter('araise.enable_turbo', $config['enable_turbo']);
```
Comment (tuxes3) [inline src/DependencyInjection/araiseTableExtension.php:25]: this is not needed

File: src/Resources/views/tailwind_2_layout.html.twig [hunk 2]
```diff
@@ -11,7 +11,7 @@
                 }))
             }) }}
         >
-            <turbo-frame id="whatwedo_table_{{ table.identifier }}" data-turbo="true" target="_top" {{ stimulus_action('@araise/table-bundle/table_select', 'tableTargetChanged', 'turbo:frame-load') }}>
+            <turbo-frame id="whatwedo_table_{{ table.identifier }}" data-turbo="{{ araise_turbo_enabled ? 'true' : 'false' }}" target="_top" {{ stimulus_action('@araise/table-bundle/table_select', 'tableTargetChanged', 'turbo:frame-load') }}>
```
Comment (tuxes3) [inline src/Resources/views/tailwind_2_layout.html.twig:14]: create another twig extension for the parameter `araise_table.enable_turbo` and use this one.

File: src/DependencyInjection/araiseTableExtension.php [hunk 3]
```diff
@@ -6,35 +6,25 @@
...
 
-        $doctrineConfig = $container->getExtensionConfig('doctrine_migrations');
```
Comment (tuxes3) [inline src/DependencyInjection/araiseTableExtension.php:30]: why did you remove this part?

**Review by tuxes3 — CHANGES_REQUESTED** [review]

File: src/DependencyInjection/araiseTableExtension.php (hunk 3 above)
Comment (Ruesa18) [reply src/DependencyInjection/araiseTableExtension.php:30]: That was an accident 😅

**Review by tuxes3 — APPROVED** [review]

**Pull request merged**